noir-projects/noir-protocol-circuits/crates/rollup-lib/src/base/base_rollup_inputs.nr

```diff
@@ -110,7 +110,8 @@ impl BaseRollupInputs {
         let protocol_public_data_update_requests = [self.build_payment_update_request()];
         let end_public_data_tree_snapshot = self.validate_and_process_public_state(protocol_public_data_update_requests);
 
-        let transaction_fee = self.kernel_data.public_inputs.compute_transaction_fee();
+        let gas_fees = self.constants.global_variables.gas_fees;
```

Comment:
Not for this PR, but should we be verifying that the global_variables.gas_fees were set correctly here?

Reply:
I believe that's handled by the merge and root rollups, right?